integration/kubernetes/confidential/agent_image.bats

```diff
@@ -140,7 +140,7 @@ assert_logs_contain() {
 teardown() {
 	# Print the logs and cleanup resources.
 	echo "-- Kata logs:"
-	sudo journalctl -xe -t kata --since "$start_date"
+	sudo journalctl -xe -t kata --since "$start_date" -n 2500
```

Comment:
side note: it has printed the msgs from kata-env that usually is not what we want to. In the future I should send a patch to filter out their message. is it a good idea?

Reply:
Yeah - that's a good idea. It's definitely a fine balance between printing enough to debug and adding too much!